hbase-server/src/test/java/org/apache/hadoop/hbase/mob/TestMobFileCleanerChore.java

```diff
@@ -92,7 +92,7 @@ public void setUp() throws Exception {
 
     HTU.startMiniCluster();
     admin = HTU.getAdmin();
-    chore = new MobFileCleanerChore();
+    chore = new MobFileCleanerChore(HTU.getHBaseCluster().getMaster());
```

Comment:
Do we still use this chore anywhere in this test?

Reply:
Good point, we don't. I've removed this field and renamed the class TestMobFileCleanupUtil.